Challenge-4/Bot_2/src/agent.spec.ts

```diff
@@ -315,68 +338,66 @@ describe("Compound test suite for lending and borrowing", () => {
 
   // Test to check if there are multiple findings for the same case above
 
-  it("returns multiple findings among multiple transactions that are not withdraw, if the utlization is below the borrow kink", async () => {
+  it("returns multiple findings among multiple transactions that are not withdraw, if the utilization is below the borrow kink", async () => {
     setupMockProvider(
       ethers.BigNumber.from("900000000000000000"),
       ethers.BigNumber.from("9600000000000000000"),
       ethers.BigNumber.from("300000000000000000"),
     );
 
-    mockProvider.setNetwork(1);
-    await initialize();
     txEvent.addTraces(
       {
         function: provideInterface.getFunction("withdraw"),
-        to: createAddress("0xc3d688B66703497DAA19211EEdff47f25384cdc3"),
+        to: mockEthUsdcTokenAddress,
         from: createAddress("0x123"),
-        arguments: ["0xc3d688B66703497DAA19211EEdff47f25384cdc3", 10],
+        arguments: [mockEthUsdcTokenAddress, 10],
       },
       {
         function: provideInterface.getFunction("withdraw"),
-        to: createAddress("0xc3d688B66703497DAA19211EEdff47f25384cdc3"),
+        to: mockEthUsdcTokenAddress,
         from: createAddress("0x123"),
-        arguments: ["0xc3d688B66703497DAA19211EEdff47f25384cdc3", 20],
+        arguments: [mockEthUsdcTokenAddress, 20],
       },
       {
         function: provideInterface.getFunction("otherFunction"),
-        to: createAddress("0xc3d688B66703497DAA19211EEdff47f25384cdc3"),
+        to: mockEthUsdcTokenAddress,
         from: createAddress("0x123"),
-        arguments: ["0xc3d688B66703497DAA19211EEdff47f25384cdc3", 10],
+        arguments: [mockEthUsdcTokenAddress, 10],
       },
     );
 
     const findings = await handleTransaction(txEvent);
+
     expect(findings.length).toBe(2);
```

Comment:
Quick reminder to not go the route of only checking the length, but the contents of the Finding itself.